ui/components/app/gas-timing/gas-timing.component.js

```diff
@@ -83,4 +164,5 @@ export default function GasTiming({ maxPriorityFeePerGas }) {
 
 GasTiming.propTypes = {
   maxPriorityFeePerGas: PropTypes.string.isRequired,
+  maxFeePerGas: PropTypes.string.isRequired,
```

Comment:
Note for myself: I will need to pass `maxFeePerGas` when calling the `GasTiming` component from Swaps: https://github.com/MetaMask/metamask-extension/blob/develop/ui/pages/swaps/fee-card/fee-card.js#L144

I will send a small PR for that once this one is merged.

Reply:
Here is a PR for that: https://github.com/MetaMask/metamask-extension/pull/11684